drivers/infiniband/hw/bnxt_re/ib_verbs.c

```diff
@@ -133,7 +133,7 @@ int bnxt_re_query_device(struct ib_device *ibdev,
 	bnxt_qplib_get_guid(rdev->netdev->dev_addr,
 			    (u8 *)&ib_attr->sys_image_guid);
 	ib_attr->max_mr_size = BNXT_RE_MAX_MR_SIZE;
-	ib_attr->page_size_cap = BNXT_RE_PAGE_SIZE_4K | BNXT_RE_PAGE_SIZE_2M;
+	ib_attr->page_size_cap = BNXT_RE_PAGE_SIZE_SUPPORTED;
```

Comment:
in oob, we seem to support only 
attr->page_size_cap = BIT_ULL(28) | BIT_ULL(21) | BIT_ULL(12);
is the plan to update oob similar to upstream?

Reply:
yes.. we can support all  all bits upto 1G in OOB also.